googletest/include/gtest/gtest.h

```diff
@@ -1781,12 +1783,24 @@ AssertionResult CmpHelperFloatingPointEQ(const char* lhs_expression,
 // Helper function for implementing ASSERT_NEAR.
 //
 // INTERNAL IMPLEMENTATION - DO NOT USE IN A USER PROGRAM.
-GTEST_API_ AssertionResult DoubleNearPredFormat(const char* expr1,
-                                                const char* expr2,
-                                                const char* abs_error_expr,
-                                                double val1,
-                                                double val2,
-                                                double abs_error);
+template <class R1, class R2, class T>
+AssertionResult FPNearPredFormat(const char* expr1, const char* expr2,
+                                 const char* abs_error_expr, const R1& val1,
+                                 const R2& val2, const T& abs_error) {
+  using namespace std;
```

Comment:
Please follow the style guide. See https://google.github.io/styleguide/cppguide.html#Namespaces